lib/beaker/ssh_connection.rb

```diff
@@ -147,12 +149,16 @@ def wait_for_connection_failure options = {}, stdout_callback = nil, stderr_call
         slept = 0
         stdout_callback.call("sleep #{wait} second(s): ")
         while slept < wait
-          sleep slept
+          sleep 1
           stdout_callback.call('.')
           slept += 1
         end
         stdout_callback.call("\n")
-        (last_wait, wait) = wait, last_wait + wait
+        if poll_interval
+          wait = poll_interval
+        else
+          (last_wait, wait) = wait, last_wait + wait
+        end
```

Comment:
@samwoods1 if I'm understanding your implementation correctly, this will poll at the same time interval if you pass the `poll_interval` parameter, or do the fibonacci backoff it's not provided. Is that correct?

I'm curious why you want to provide polling at a set interval in addition to the fibonacci backoff that already exists. This code in general (historically, unrelated to your changes) seems like a reproduction of the [repetition.rb](https://github.com/puppetlabs/beaker/blob/master/lib/beaker/shared/repetition.rb) code, and so my immediate response would be that if this code is touched, it should be refactored to use the shared repetition code. Having both cases supported seems to be a block to that refactor, so I'm curious about what's motivating you to need the static polling interval.